README.md

```diff
@@ -8,6 +8,10 @@
 
 An asynchronous effect monad and threading model for PureScript.
 
+`aff` is in the same space as JavaScript Promises, but with additional support for cancellation (async exceptions) and resource management (`bracket`). It’s cooperative multi-tasking which yields on async constructors (`makeAff`).
```

Comment:
I find this a little easier to parse, but no need to apply the suggestion if you feel differently!

```suggestion
`aff` is in the same space as JavaScript Promises, but with additional support for cancellation (async exceptions) and resource management (`bracket`). It provides cooperative multi-tasking which yields on async constructors (`makeAff`).
```